src/lib/updating-element.ts

```diff
@@ -15,27 +15,28 @@
 /**
  * Converts property values to and from attribute values.
  */
-export interface AttributeSerializer<T = any> {
+export interface ComplexAttributeConverter<Type = any, TypeHint = any> {
 
   /**
    * Deserializing function called to convert an attribute value to a property
```

Comment:
IMO `Deserializing` and `Deserializing` is a confusing terminology.
Is it still needed after this change ?

Reply:
Done.